packages/angular/src/ng-add-setup-project/rules/start.ts

```diff
@@ -0,0 +1,30 @@
+import {normalize} from '@angular-devkit/core';
+import {Rule, SchematicContext, Tree} from '@angular-devkit/schematics';
+import {CoveoSchema} from '../../schema';
+
+/**
+ * Create a npm script in the project root to start both the Angular app and the proxy server
+ */
+export function configureStartCommand(_options: CoveoSchema): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    const packageJsonBuffer = tree.read(normalize('./package.json'));
+    if (packageJsonBuffer === null) {
```

Comment:
Actually no. I'll remove it. there is no point in that check. if something breaks, it will happen in the try catch

Reply:
Oh now I remember. It returns a TS error otherwise
![image](https://user-images.githubusercontent.com/12199712/119545120-74432680-bd60-11eb-8a75-5950157a8bc3.png)